prow/plugins/trigger/generic-comment.go

```diff
@@ -29,6 +31,8 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
+var testHelpRe = regexp.MustCompile(`(?m)^/test\s*(\?|-ls)$`)
```

Comment:
I'm not sure `/test ?` or `/test -ls` is the best command for this. The first is fine-ish, the second is IMO unnecessarily technical and unix-y.  I'm thinking if something more human oriented, like `/list-tests` would be better?

Reply:
Thank you for the review. I have updated the PR to use `/test ?` and `/list-tests` as help commands.